app/assets/javascripts/govuk_publishing_components/components/add-another.js

```diff
@@ -0,0 +1,159 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function AddAnother (module) {
```

Comment:
Is "Add Another" the best name for this component? It's consistent with the MoJ naming which has a certain amount of value, but I feel that it isn't hugely descriptive.

Reply:
I'm pro sticking with MoJ unless we can come up with something significantly better